pkg/ranger/reader.go

```diff
@@ -19,6 +19,24 @@ type Ranger interface {
 	Range(offset, length int64) io.ReadCloser
 }
 
+// A RangerCloser is a Ranger that must be closed when finished
+type RangerCloser interface {
+	Ranger
+	Close() error
```

Comment:
Why don't you just refer to the `io.Closer` interface?

Reply:
because i'm dumb